plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SnappyTickReader.java

```diff
@@ -115,40 +116,17 @@ private int extractTestNumber(String testfile) {
         return -1;
     }
 
-    private int cweLookup(String checkerKey) {
-        switch (checkerKey.trim()) {
-            case "1004":
-                return 614; // HTTPOnly Flag Not Set For Cookies:insecure cookie use
-            case "614":
-                return 614; // Cookie not Sent Over SSL:insecure cookie use
-            case "78":
-                return 78; // command injection
-            case "89":
-                return 89; // SQL injection
-            case "755":
-                return 755; // SQL Exception Vulnerability:Info Leak
+    private CweNumber cweLookup(String checkerKey) {
+        String cwe = checkerKey.trim();
+
+        switch (cwe) {
             case "258":
-                return 000; // "Use an empty string as a password"
-            case "20":
-                return 20; // "Input Validation Issue or Input Validation Required"
-            case "79":
-                return 79; // Malicious Scripting Attacks and xss
+                return CweNumber.DONTCARE;
             case "73":
-                return 22; // Path Manipulation: path traversal
             case "538":
-                return 22; // File Disclosure Vulnerability:path traversal
-            case "330":
-                return 330; // Use of java.util.Random generator function:weak random
-            case "327":
-                return 327; // Broken Cryptography or
-                // Weak Encryption Insecure Mode of Operation:weak encryption
-            case "328":
-                return 328; // Broken Hashing algorithm
-            default:
-                System.out.println(
-                        "Found unrecognized vulnerability type in Snappy Tick results: "
-                                + checkerKey);
+                return CweNumber.PATH_TRAVERSAL;
         }
-        return 0;
+
```

Comment:
Can we restore what we had so we can flag reported CWEs not yet mapped?

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379